apps/web/src/app/_components/form/submission-failure.tsx

```diff
@@ -0,0 +1,24 @@
+import Link from "next/link";
+
+import { Button } from "@cooper/ui/button";
+
+export function SubmissionFailure() {
+  return (
+    <div className="flex flex-col border-2">
+      <div className="z-10 -mb-4 h-4 w-full rounded-t-xl bg-gradient-to-r from-rose-400 via-yellow-400 to-blue-600" />
+      <div className="flex h-[80vh] w-full flex-col items-center justify-center space-y-6 rounded-xl bg-white px-8 py-32 text-cooper-blue-600">
+        <div className="text-5xl font-bold">
+          Whoops, your review is invalid...
+        </div>
+        <div className="max-w-3xl text-center text-2xl">
+          IM SORRY IT DOES NOT WORK
```

Comment:
See if you can display a more meaningful message based on the error that you get